cmd/ipfs/goreq.go

```diff
@@ -0,0 +1,3 @@
+// +build !go1.5		
+
+`IPFS needs to be built with go version 1.5 or greater`
```

Comment:
the go version based build tags are incremental, so `!go1.5` implies 'dont build this for anything 1.5 or greater'